developer-docs-site/docs/sdks/aptos-sdk-overview.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Aptos SDK"
+slug: "aptos-sdk-overview"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos SDK
+
+This document describes the main features and components of the Aptos SDK.
+
+The Aptos SDK provides APIs and interfaces you can use to interact with the Aptos Blockchain by connecting to the Aptos REST API. The REST API is the means for sending your transaction to the Aptos Blockchain, and reading the blockchain's state. 
+
+:::tip
+To see how to use the Aptos SDK, see the tutorial [Transactions with Typescript SDK](/sdks/transactions-with-ts-sdk). Also see the [Aptos SDK docs](https://aptos-labs.github.io/ts-sdk-doc/).
+:::
+
+See below a high-level architecture diagram of the Aptos Typescript SDK.
+
+<ThemedImage
+alt="Signed Transaction Flow"
+sources={{
+    light: useBaseUrl('/img/docs/ts-sdk-light.svg'),
+    dark: useBaseUrl('/img/docs/ts-sdk-dark.svg'),
+  }}
+/>
+
+## Key SDK features
+
+The following are a few key features of the Aptos SDK:
+
+- **Key generation:** The Aptos SDK provides convenient methods for generating [Ed25519](https://ed25519.cr.yp.to/) key pairs. The Ed25519 public keys can be used to derive the chain account addresses, while the private keys should be kept secret for transaction signing. See the [class TransactionBuilderEd25519](https://aptos-labs.github.io/ts-sdk-doc/classes/TransactionBuilderEd25519.html).
```

Comment:
maybe private not secret? secret makes it feel like you don't tell anyone about its existence :P